### DISCUSSION TITLE
Super+Enter does not open a new terminal

### DISCUSSION THREAD

**Discussion by sedrik** [regolith-linux/regolith-desktop#968] [General]
Hi

I'm running a fresh Ubuntu install (23.04 as it is the latest supported version).

When I start Regolith (installed using the instructions on the regolith-desktop homepage) I can't launch a new terminal (or anything at all basically).

After some digging around I noticed that my system is missing the file that should setup the shortcut (https://github.com/regolith-linux/regolith-wm-config/blob/main/usr/share/regolith/common/config.d/15_base_launchers). I have the regolith-wm-config package installed which from what I can tell should contain that file.

Did I do something wrong?

**Comment by sedrik** [comment]
I did a complete re-install using the 23.10 iso even though according to the homepage it is not supported and that installed the keymappings so now it works.

Are the packages for 23.04 broken?

**Comment by kgilmer** [comment]
Hi @sedrik , yes that package (`regolith-wm-base-launchers`) should have been installed along with `regolith-wm-config`.  Here is the dependency delcaration: https://github.com/regolith-linux/regolith-wm-config/blob/main/debian/control#L16.  Because it is a soft dependency, if that package or child dependency has some conflict, `apt` will choose not to install it but will continue with the general installation because it is "soft".  That may be why the package was not brought in.  But, as you mention that you started as a fresh install, that's unlikely.  There are `apt` logs that might offer more info to what's going in, in `/var/log/apt`.

`23.04` (lunar) has been deprecated upstream, we are not building updates to that version anymore.

**Comment by sedrik** [comment]
As I said I have done a re-install from 23.10 and that works just fine now so I don't have the logs any longer.

I would still point out that the homepage has 23.04 as it's latest Ubuntu release still so the experience for new users is not great if they run into this :)

**Reply by kgilmer** [reply]
Oh, sorry I missed the version detail.  Yes, we're in a weird spot right now, `lunar` has been deprecated but we have yet to release a beta for `mantic` support and so `focal` and `jammy` (LTS) releases are all that are available at the moment.  You seem to have unknowningly joined the (pre) beta test of Regolith 3.1 (you can check in the file `/etc/regolith/version`) :smile: .  We hope to officially ship the first beta soon, but are working through some issues in our GNOME integration.  You can learn more in the 3.1 release notes PR: https://github.com/regolith-linux/regolith-desktop-website/pull/38

**Reply by kgilmer** [reply]
As a sanity check, what do you have for your regolith `apt` URL (`/etc/apt/sources.list.d/regolith.list`)?

**Comment by sedrik** [comment]
And thanks for all the hard work you are putting in to this project btw :)

**Comment by sedrik** [comment]
Hi agin, sorry for the late reply. I'm not on 3.1 just yet it seems. But things are up and running and any issues I still have is probably more related to Ubuntu and snap. Opening the gnome-control-center takes forever for example, but my understanding is that it is related to the xdg-desktop-portal.

```
❯ cat /etc/regolith/version
REGOLITH_VERSION=3.0
```

```
> cat /etc/apt/sources.list.d/regolith.list
deb [arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] https://regolith-desktop.org/release-3_0-ubuntu-lunar-amd64 lunar main
```
